### ISSUE TITLE
Bazel file detection defeated by MacVim's filetype.vim

### ISSUE THREAD

**Issue by patspam** [bazelbuild/vim-ft-bzl#8]
After installing this plugin and opening a *.bzl file in MacVim, I get:

``` vim
:verbose set ft?
```

```
filetype=conf
Last set from /Applications/MacVim.app/Contents/Resources/vim/runtime/filetype.vim
```

I'm currently working around this via:

``` vim
au BufRead,BufNewFile *.bzl,BUILD,WORKSPACE set ft=bzl
```

(Note: I'm using `set ft=` instead of `setf` in order to defeat MacVim's filetype.vim)

**Comment by dbarnett** [comment]
Yeah, reasonable to change it to `set ft=` instead of `setf`. No point merging that change into vim since the builtin version has control over precedence.

BTW, I remember that aggressive conf rule in older versions of vim but I don't see it anymore in recent versions.
